Source/WebCore/dom/ScriptExecutionContext.h

```diff
@@ -215,6 +216,8 @@ class ScriptExecutionContext : public SecurityContext, public TimerAlignment {
     WEBCORE_EXPORT void refAllowingPartiallyDestroyed();
     WEBCORE_EXPORT void derefAllowingPartiallyDestroyed();
 
+    WEBCORE_EXPORT bool requiresScriptExecutionTelemetry(ScriptTelemetryCategory);
```

Comment:
Is this function supposed to be added in this patch? It’s not used.

Reply:
Oh, it'll be used in the next patch, which will adopt this mechanism (along with the new enum type).